us-geological-survey.csl

```diff
@@ -96,7 +96,7 @@
       </group>
       <choose>
         <if type="report">
-          <group prefix=" " delimiter=", " suffix=".">
+          <group prefix=" " delimiter=", " suffix=".,">
```

Comment:
We'd want this handled with a delimiter, so you don't have a report without URL ending on `.,.` which currently could happen. Remove the suffix entirely (including the period). 
Then remove `</group>` from line 110 and place it after line 128 instead. Please test to make sure I haven't gotten something mixed up.